.github/workflows/nightly-trigger.yml

```diff
@@ -1,3 +1,4 @@
+# base : aripitek09@gmail.com
```

Comment:
this PR should either have a Jira in the title or indicate it is a hotfix as per the [process](https://cwiki.apache.org/confluence/display/FLINK/Pull+Request+Management).

This looks strange to add in what looks like an individuals gmail to a github action. Please could you add some description in the PR or Jira as to why this is required.